ods_ci/tests/Resources/Files/llm/vllm/vllm_servingruntime.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: serving.kserve.io/v1alpha1
+kind: ServingRuntime
+metadata: 
+  name: kserve-vllm
+  namespace: vllm-gpt2
+spec: 
+  annotations: 
+    sidecar.istio.io/inject: "true"
+    sidecar.istio.io/rewriteAppHTTPProbers: "true"
+    serving.knative.openshift.io/enablePassthrough: "true"
+    opendatahub.io/dashboard: "true"
+    openshift.io/display-name: "vLLLM Openai entry point"
+    serving.kserve.io/enable-prometheus-scraping: "false"
+    serving.kserve.io/enable-metric-aggregation: "true"
+    prometheus.io/port: '8080'
+    prometheus.io/path: "/metrics/"
+  multiModel: false
+  supportedModelFormats: 
+    - name: vLLM
+      autoSelect: true
+  containers: 
+    - name: kserve-container
+      #image: kserve/vllmserver:latest
+      image: quay.io/wxpe/tgis-vllm:release.74803b6
```

Comment:
Where is the registry supposed to be? is `wxpe` expected?

Reply:
as of now we do not have official image so we are testing with this image which has fix for multi-ggpu